### PR TITLE
Delete the deprecated Gatsby Image Component

### PR DESCRIPTION
**Pull request by olavea**
Yo-Ho-Ho and a bottle of GOOO!!
🍾

### CODE REVIEW

File: packages/gatsby-transformer-cloudinary/api/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-  getFluidImageObject,
-} = require('../gatsby-image/get-image-objects');
+// delete this file then? Asks Cap'n Ola Vea Jr.
 
-// Runtime APIs for fetching remote images on Cloudinary
-module.exports = {
-  getFixedImageObject,
-  getFluidImageObject,
-};
+// const {
```
Comment (raae) [inline packages/gatsby-transformer-cloudinary/api/index.js]: nitpick(blocking): remove code instead of commenting it out

**Review by raae — CHANGES_REQUESTED** [review]
praise: great job so far and awesome commit history 🎉

---

suggestion (blocking): remove the manual examples as they use `gatsby-image`, but not `gatsby-plugin-image`
- [x] delete `manual-fixed` and `manual-fluid` from the examples
- [x] delete the `manual` page
- [x] remove the link to the `manual` page in the `header` component

---

next step: delete gatsby-image documentation in the readme:
- [x] section on gatsby-image
- [x] gatsby-image only configurations in from the configuration section

File: packages/gatsby-transformer-cloudinary/api/index.js (hunk 1 above)
Comment (olavea) [reply packages/gatsby-transformer-cloudinary/api/index.js]: 😺

**Comment by olavea** [comment]
How do we uninstall gatsby-image@3.11.0?

**Comment by raae** [comment]
> How do we uninstall gatsby-image@3.11.0?

You remove it from the `package.json` files as in remove the line. Then do a new `yarn install` in root to update `yarn.lock`.

**Review by raae — COMMENTED** [review]
Yes delete `getFixedImage` etc. if they are imported from the `gatsby-image` folder you have deleted. 

You can delete the whole `api` folder I guess.

File: packages/gatsby-transformer-cloudinary/api/index.js (hunk 1 above)
Comment (olavea) [reply packages/gatsby-transformer-cloudinary/api/index.js]: 😺

**Comment by raae** [comment]
Looking good, will give it a proper review later.

File: packages/gatsby-transformer-cloudinary/README.md [hunk 2]
```diff
@@ -341,134 +319,14 @@ Go to the [Gatsby Plugin Image Docs](https://www.gatsbyjs.com/docs/reference/bui
...
-| `width`  | `Int` | `400`   | The width that the image should display at.  |
-
-### Arguments for `fluid`
-
-| argument   | type  | default                     | description                         |
-| ---------- | ----- | --------------------------- | ----------------------------------- |
-| `maxWidth` | `Int` | Original width of the image | The maximum width for fluid images. |
-
+<!--
+Should we not delete "aspect ratios"
```
Comment (raae) [inline packages/gatsby-transformer-cloudinary/README.md]: Yes, you should delete the note on aspect ratio @olavea

**Review by raae — CHANGES_REQUESTED** [review]
suggestion: delete `<h3>gatsby-plugin-image</h3>` from examples

They no longer make sense

**Review by raae — COMMENTED** [review]
praise: good job on the `<h3>gatsby-plugin-image</h3>` removal

File: packages/gatsby-transformer-cloudinary/README.md (hunk 2 above)
Comment (olavea) [reply packages/gatsby-transformer-cloudinary/README.md]: see [fce1446](https://github.com/raae/gatsby-transformer-cloudinary/pull/8/commits/fce144627ade4927875a5bae354d08d715550172)

**Pull request closed without merging**